modules/ols-1-0-known-issues.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-1-0-known-issues_{context}"]
+= Known issues
+
+The following issues are identified with {ols-official} 1.0:
+
+* For {ocp-product-title} 4.17 and later, the {ols-short} icon disappears when you click *Create Namespace* or *Create Project* from one of the following locations in the {ocp-product-title} web console:
+
+** *Administration -> Namespaces* 
+** *Home -> Projects -> Create Project*
+** The *Project* drop-down menu located at the top of most pages
++
+Workaround: Refresh the web browser and the {ols-short} icon appears. link:https://issues.redhat.com/browse/OLS-1815[OLS-1815].
+
+* Changing the value of the `quota` parameter in the `spec.ols.quota_handlers.conf` specification of the `OLSConfig` custom resource file does not take effect until the currently defined quota period expires.
```

Comment:
Changing the value of the **initialQuota** in the **spec.ols.quotaHandlersConfig.limitersConfig** (...)

Reply:
@xrajesh please confirm this one as well.

Reply:
I just pushed this change